js/ui/windowManager.js

```diff
@@ -1458,7 +1456,7 @@ const WindowManager = new Lang.Class({
             }
         }
 
-        if (this._codingManager) {
+        if (global.settings.get_boolean('enable-behind-the-screen')) {
```

Comment:
Discussed this with Cosimo. The setting is not user visible, chances that it is modified are small.